### PR TITLE
Sign-Up Test

### PR DESCRIPTION
**Pull request by ranuccimariano**
## Summary

- Created test for `/users` controller:
  - Test succeeds with valid parameters.
  - Test fails with invalid parameters or with no parameters.
  - Created "test-migrations" script to automatically run migrations in test environment database.

## Trello Card

https://trello.com/c/ajSCYBbX

### CODE REVIEW

File: test/app/controllers/users.test.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+const request = require('supertest');
```
Comment (molinajulian) [inline test/app/controllers/users.test.js]: Add whitespace line after supertest

File: test/app/controllers/users.test.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+const app = require('../../../app');
+
+describe('Create User', () => {
+  describe('With valid params', () => {
```
Comment (molinajulian) [inline test/app/controllers/users.test.js]: Use describe to join test cases, for example : ' POST /users', 'Successful cases', 'Failure cases'

File: test/app/controllers/users.test.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    it('should create a new user', async () => {
+      const res = await request(app)
+        .post('/users')
+        .send({
+          first_name: 'Alberto',
+          last_name: 'Alvarez',
+          email: 'alberto.alvarez@wolox.com.ar',
+          password: 'albertAlv85'
+        });
+      expect(res.statusCode).toEqual(201);
```
Comment (molinajulian) [inline test/app/controllers/users.test.js]: Would you check more fields please? Apply this comment for all test

File: test/app/controllers/users.test.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    it("shouldn't create a new user", async () => {
+      const res = await request(app)
+        .post('/users')
+        .send({
+          first_name: 'Alberto',
+          last_name: 'Alvarez',
+          email: 'alberto.alvarez@gmail.com',
+          password: 'albertAlv85'
+        });
+      expect(res.statusCode).toEqual(500);
```
Comment (molinajulian) [inline test/app/controllers/users.test.js]: Same

File: test/app/controllers/users.test.js [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+    it("shouldn't create a new user", async () => {
+      const res = await request(app)
+        .post('/users')
+        .send({
+          first_name: 'Alberto',
+          last_name: 'Alvarez',
+          email: 'alberto.alvarez@wolox.com.ar',
+          password: 'albalv'
+        });
+      expect(res.statusCode).toEqual(500);
```
Comment (molinajulian) [inline test/app/controllers/users.test.js]: Same

File: test/app/controllers/users.test.js [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+    it("shouldn't create a new user", async () => {
+      const res = await request(app)
+        .post('/users')
+        .send({
+          first_name: '',
+          last_name: '',
+          email: '',
+          password: ''
+        });
+      expect(res.statusCode).toEqual(500);
```
Comment (molinajulian) [inline test/app/controllers/users.test.js]: Empty params is a 400 or 422 but not 500

File: app/controllers/users.js [hunk 7]
```diff
@@ -32,7 +28,10 @@ exports.postUser = (req, res, next) => {
       hashPassword(password)
         .then(() =>
           createUser(firstName, lastName, email, password)
-            .then(logger.info(`Usuario creado para: ${firstName} ${lastName}`))
+            .then(user => {
+              logger.info(`Usuario creado para: ${firstName} ${lastName}`);
```
Comment (molinajulian) [inline app/controllers/users.js]: The logs always in English

File: test/app/controllers/users.test.js [hunk 8]
```diff
@@ -1,74 +1,117 @@
...
-      const res = await request(app)
-        .post('/users')
-        .send({
-          first_name: 'Alberto',
-          last_name: 'Alvarez',
-          email: 'alberto.alvarez@wolox.com.ar',
-          password: 'albertAlv85'
-        });
-      expect(res.statusCode).toEqual(201);
+describe('POST /users', () => {
```
Comment (molinajulian) [inline test/app/controllers/users.test.js]: Great work! Keep it up :clap:

File: app/controllers/users.js (hunk 7 above)
Comment (ranuccimariano) [reply app/controllers/users.js]: Okey :+1:

File: test/app/controllers/users.test.js (hunk 8 above)
Comment (ranuccimariano) [reply test/app/controllers/users.test.js]: Thanks :+1:

**Pull request merged**